recipes/andreasbuhr-cppcoro/all/conanfile.py

```diff
@@ -91,18 +93,14 @@ def package(self):
         tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
 
     def package_info(self):
-        self.cpp_info.filenames["cmake_find_package"] = "cppcoro"
-        self.cpp_info.filenames["cmake_find_package_multi"] = "cppcoro"
         self.cpp_info.names["cmake_find_package"] = "cppcoro"
         self.cpp_info.names["cmake_find_package_multi"] = "cppcoro"
-
-        comp = self.cpp_info.components["cppcoro"]
-        comp.names["cmake_find_package"] = "cppcoro"
-        comp.names["cmake_find_package_multi"] = "cppcoro"
-
+        self.cpp_info.libs = ["cppcoro"]
+        if self.settings.os in ["Linux", "FreeBSD"]:
+            self.cpp_info.system_libs.append("pthread")
         if self.settings.compiler == "Visual Studio":
-            comp.cxxflags.append("/await")
+            self.cpp_info.cxxflags.append("/await:strict")
```

Comment:
```suggestion
            if tools.Version(self.settings.compiler.version) <= "15"):
                self.cpp_info.cxxflags.append("/await")
            self.cpp_info.cxxflags.append("/await:heapelide")
```

For the first approach I mentioned.